recipes/theseus-ai/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set name = "theseus-ai" %}
+{% set version = "0.1.3" %}
+{% set build = 0 %}
+
+{% set processor = "cpu" if cuda_compiler_version == "None" else "gpu" %}  # [not osx]
+{% set processor = "cpu" %}  # [osx]
+
+# Prioritize gpu build if cudatoolkit can be installed (through __cuda virtual package)
+{% set build = build + 100 if processor == "gpu" else build %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/theseus-ai-{{ version }}.tar.gz
+  sha256: 150702236782c76815ad349d2089ef801469a9a8073610da77fe4b89a11ad5d1
+
+build:
+  number: {{ build }}
```

```suggestion
  number: 0
```

Don't weigh down via build number but use the following pattern: https://github.com/conda-forge/tensorflow-feedstock/blob/f73deb53c2b170ca3116b4faee9e58c4409b6717/recipe/meta.yaml#L132

---

Thanks - I hadn't used track_features before :)